### DISCUSSION TITLE
Icons pi pi-check not showing

### DISCUSSION THREAD

**Discussion by SumunaTJ** [primefaces/community#2618] [PrimeIcons]
<ToggleButton onLabel="" offLabel="" onIcon="pi pi-check" offIcon="pi pi-times" checked={checked} onChange={(e) => setChecked(e.value)} />

my version
"primeicons": "^6.0.1",
"primereact": "^10.0.9",

its show like that, how can I fix it
![Screenshot 2024-09-02 142204](https://github.com/user-attachments/assets/961f6f11-8043-4778-ac14-8a92930a5f05)

**Comment by abdullasirajudeen** [comment]
![image](https://github.com/user-attachments/assets/6a59ea4c-6d8d-4291-a973-4bf4b32f00db)

![image](https://github.com/user-attachments/assets/eb26708c-0ecf-4356-944b-f827c05dea18)

`<p-button icon="pi pi-check"></p-button>`

Its working fine
